MdePkg/Include/Protocol/Tcg2Protocol.h

```diff
@@ -298,6 +309,7 @@ struct tdEFI_TCG2_PROTOCOL {
   EFI_TCG2_GET_CAPABILITY                        GetCapability;
   EFI_TCG2_GET_EVENT_LOG                         GetEventLog;
   EFI_TCG2_HASH_LOG_EXTEND_EVENT                 HashLogExtendEvent;
+  EFI_TCG2_LOG_EVENT                             LogEvent;

```

Comment:
See comment in the other PR about this function needing to go to the end. Also, making `EFI_TCG2_PROTOCOL` users aware of the function's presence. Consideration of a V2 protocol if necessary.